src/landscape/components/LandscapeDefaultHomeCard.js

```diff
@@ -15,50 +15,39 @@
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
 import React from 'react';
-import { useTranslation } from 'react-i18next';
-import { Alert, Box, Divider, Typography } from '@mui/material';
-import CardActionRouterLink from 'common/components/CardActionRouterLink';
+import { Trans, useTranslation } from 'react-i18next';
+import { Box, Typography } from '@mui/material';
+import RouterLink from 'common/components/RouterLink';
 import HomeCard from 'home/components/HomeCard';
 
 const LandscapeDefaultHomeCard = () => {
   const { t } = useTranslation();
-
+  const landscapeImage = require(`assets/${t(`landscape.img.src`)}`);
   return (
     <HomeCard
-      aria-labelledby="landscapes-default-title"
-      sx={{ flexDirection: 'column' }}
+      title={t('landscape.home_title')}
+      ariaLabel={t('landscape.aria_label_default')}
```

Comment:
Issue: This is an id, so it seems it should be
```suggestion
      ariaLabel="landscapes-default-title"
```
Also, check the other comments for the rename from `ariaLabel` to `ariaLabelledBy`